app/src/main/java/com/android/partagix/model/UserViewModel.kt

```diff
@@ -49,8 +49,7 @@ class UserViewModel(
 
     if (userID == null) {
       database.getUser("XogPd4oF1nYc6Rag6zhh") { updateUIState(it) }
-    } else if (userID != "" &&
-        false) { // TODO: remove false when logged in users are in the database
+    } else if (userID != "") {
       database.getUser(userID) { updateUIState(it) }
     } else {
       database.getUser("XogPd4oF1nYc6Rag6zhh") { updateUIState(it) }
```

Comment:
Seems like we'll never go to this else so I would remove it

Reply:
You're right ! I've changed the if-else condition to make it clearer in  `setUserToCurrent()`